httpclient5-testing/src/main/java/org/apache/hc/client5/testing/auth/BasicAuthTokenExtractor.java

```diff
@@ -49,9 +47,9 @@ public String extract(final String challengeResponse) throws HttpException {
                 final String s = challengeResponse.substring(i + 1).trim();
                 try {
                     final byte[] credsRaw = s.getBytes(StandardCharsets.US_ASCII);
-                    final BinaryDecoder codec = new Base64();
+                    final Base64 codec = new Base64();
                     return new String(codec.decode(credsRaw), StandardCharsets.US_ASCII);
-                } catch (final DecoderException ex) {
```

Comment:
Although `BinaryDecoder.decode` declares this exception, the Base64 subclasses did not actually ever throw the exception. I've left the handling in place as a reminder of how invalid inputs can be handled.